backend/database/models/infra/locations.py

```diff
@@ -0,0 +1,123 @@
+    "NJ": {"name": "New Jersey",               "capital": "Trenton"},
+    "NM": {"name": "New Mexico",               "capital": "Santa Fe"},
+    "NY": {"name": "New York",                 "capital": "Albany"},
+    "NC": {"name": "North Carolina",           "capital": "Raleigh"},
+    "ND": {"name": "North Dakota",             "capital": "Bismarck"},
+    "OH": {"name": "Ohio",                     "capital": "Columbus"},
+    "OK": {"name": "Oklahoma",                 "capital": "Oklahoma City"},
+    "OR": {"name": "Oregon",                   "capital": "Salem"},
+    "PA": {"name": "Pennsylvania",             "capital": "Harrisburg"},
+    "RI": {"name": "Rhode Island",             "capital": "Providence"},
+    "SC": {"name": "South Carolina",           "capital": "Columbia"},
+    "SD": {"name": "South Dakota",             "capital": "Pierre"},
+    "TN": {"name": "Tennessee",                "capital": "Nashville"},
+    "TX": {"name": "Texas",                    "capital": "Austin"},
+    "UT": {"name": "Utah",                     "capital": "Salt Lake City"},
+    "VT": {"name": "Vermont",                  "capital": "Montpelier"},
+    "VA": {"name": "Virginia",                 "capital": "Richmond"},
+    "WA": {"name": "Washington",               "capital": "Olympia"},
+    "WV": {"name": "West Virginia",            "capital": "Charleston"},
+    "WI": {"name": "Wisconsin",                "capital": "Madison"},
+    "WY": {"name": "Wyoming",                  "capital": "Cheyenne"},
+    "DC": {"name": "District of Columbia",     "capital": "Washington"},
+    "AS": {"name": "American Samoa",           "capital": "Pago Pago"},
+    "GU": {"name": "Guam",                     "capital": "Hagatna"},
+    "MP": {"name": "Northern Mariana Islands", "capital": "Saipan"},
+    "PR": {"name": "Puerto Rico",              "capital": "San Juan"},
+    "VI": {"name": "U.S. Virgin Islands",      "capital": "Charlotte Amalie"},
+}
+
+
+class Place(StructuredNode):
+    """
+    Base class for all places. Adds the 'Place' label to all subclasses.
+    """
+
+    uid = UniqueIdProperty()
+    name = StringProperty(required=True)
+    coordinates = PointProperty(crs="wgs-84")
+
+    def __repr__(self):
+        return f"<Place {self.name}>"
+
+
+class StateNode(Place):
+    abbreviation = StringProperty(required=True, unique_index=True)
+
+    # Relationships
+    capital = RelationshipTo("CityNode", "HAS_CAPITAL", cardinality=ZeroOrOne)
+    counties = RelationshipTo("CountyNode", "HAS_COUNTY")
+
+    def __repr__(self):
+        return f"<State {self.name}>"
+
+
+class CountyNode(Place):
+    fips = StringProperty(required=True, unique_index=True)
+
+    # Relationships
+    state = RelationshipTo("StateNode", "WITHIN_STATE")
+    cities = RelationshipTo("CityNode", "HAS_CITY")
+
+    def __repr__(self):
+        return f"<County {self.name}>"
+
+
+class CityNode(Place):
+    population = StringProperty()
```

Comment:
Missed this before, but this should be an `IntegerProperty`. I'll fix this in the loader as well.